webrtc.html

```diff
@@ -8225,9 +8231,43 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
             <code>"new"</code>.</p>
           </li>
           <li>
+            <p>Let <var>transport</var> have a <dfn>[[\MaxMessageSize]]</dfn>
+            internal slot and run the steps labeled <a>update the data max
+            message size</a> to initialize it.</p>
+          </li>
+          <li>
             <p>Return <var>transport</var>.</p>
           </li>
         </ol>
+        <p>To <dfn>update the data max message size</dfn> of an <code>
+        <a>RTCSctpTransport</a></code> run the following
+        steps:</p>
+        <ol>
+          <li>
+            <p>Let <var>transport</var> be the <code><a>RTCSctpTransport</a>
+            </code> object to be updated.</p>
+          </li>
+          <li>
+            <p>Let <var>remoteMaxMessageSize</var> be the value of the
+            "max-message-size" SDP attribute read from the remote description,
```

Comment:
Do we need to add *from the SCTP transport's SDP media section* or is this clear to the reader?

Reply:
Most of our references with specific section numbers are automatically generated to not get out of sync with the external document. But there are "hard-coded" one too, so I guess it woudn't be too bad to add a reference to a top-level section here.